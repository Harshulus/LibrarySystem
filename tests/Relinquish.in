add Collection SampleCollection.csv
add member asvi
add member nic
rent 100000 111111
rent 100000 111112
rent 100000 111115
relinquish 100001 111111
relinquish 100000 111111
