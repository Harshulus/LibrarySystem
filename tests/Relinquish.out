user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Unable to return book.

user: Success.

